service/controller/version_bundle.go

```diff
@@ -1,4 +1,4 @@
-package v25
+package controller
```

Comment:
This should be moved to `pkg/project`. See https://github.com/giantswarm/kvm-operator/blob/master/pkg/project/version_bundle.go